types.go

```diff
@@ -4,26 +4,26 @@ import (
 	"time"
 )
 
-type status string
+type Status string
```

Comment:
Add a comment to the (now exported) type describing it. This error will show up when running `$ gometalinter`, or `$ golint types.go`.

```suggestion
// Status represents the result of a single a JUnit testcase. Indicates if a
// testcase was run, and if it was successful.
type Status string
```